code/modules/mob/living/carbon/human/life.dm

```diff
@@ -32,6 +32,12 @@
 		return
 	//Update our name based on whether our face is obscured/disfigured
 	name = get_visible_name()
+	//makes sure admins are adminning instead of having fun grrr
+	if(ckey == "Silicons" || "CygnusB" || "Putnam" || "R3dtaile" || "DeltaFire" || "R0-adic" || "Xantholne" || "Trojancoyote" || "Dzahlus" || "NicoDeimos" || "Bhijn" ) // Don't know if nico is still an admin, don't have an admin list anywhere so feel free to add if there's people missing
```

Comment:
Good PR, but keep in mind syntax, as this if statement will currently always be true.
add the ckey == xyz for the keys it's missing from (all except Silicons) and this should be good.

Reply:
Oh based putnam already commrnted while I was writing

Reply:
putnam better sorry 👋